fuels-abi-cli/Cargo.toml

```diff
@@ -15,4 +15,4 @@ fuels-core = { version = "0.7.0", path = "../fuels-core" }
 hex = "0.4"
```

Comment:
I feel like `fuels-contract/core` should be replaced by `fuels-rs`? Similarly to other manifests

Reply:
No, that doesn't seem to be the way it's done; when referring to local dependencies, you can point to them directly. [See this in ethers-rs for a good example](https://github.com/gakonst/ethers-rs/blob/master/ethers-signers/Cargo.toml#L17).

Reply:
Oh nvm, just realized the only instance where you did this was in the special `fuels-abigen-macro` repo, my bad